### DISCUSSION TITLE
Support the community

### DISCUSSION THREAD

**Discussion by zottce** [cleolibrary/CLEO-Redux#9] [Ideas]
Hello!

I would like to ask you to open the source code for all. It's nice to have another GTA relative project written in Rust.
I guess this project and its source code can be the next step to the future of Rust and GTA.
Also, I see a possibility to make a WASM engine based on your project.

Give us some love!

**Comment by x87** [comment]
Hi,

I don't have plans on making it open source, unless I decide to step away from development and hand it over to another person.

I'm open to discussing any kinds of SDK or plugin system to extend CR functionality.

**Reply by nils-kt** [reply]
> I decide to step away from development and hand it over to another person


That's another reason to make it open source. Why hide source with potential for the whole GTA community?
